src/libs/vg_lite_driver/lv_vg_lite_hal/vg_lite_platform.h

```diff
@@ -0,0 +1,11 @@
+#ifndef VG_LITE_PLATFORM_H
+#define VG_LITE_PLATFORM_H
+
+#include "../../../lv_conf_internal.h"
+#if LV_USE_VG_LITE_DRIVER
+
+    #include "lv_vg_lite_platform.h"
```

Comment:
`lv_vg_lite_platform.h` is not too long. What about moving its content here directly?

Reply:
I agree, the initial naming idea was to keep the consistency to the other lvgl vglite things (except driver) which receive the lv_vg_lite_ prefix.